sami2py/_core.py

```diff
@@ -282,9 +269,27 @@ def run_model(tag='model_run', lat=0, lon=0, alt=300, year=2018, day=1,
 
 
 def _generate_drift_info(fejer, ExB_drifts=None):
-    """Generates the information regarding the ExB drifts used by the model.
+    """Generate the information regarding the ExB drifts used by the model.
+
     This information is later stored in the namelist file for SAMI2
+
+    Parameters
+    ----------
+    fejer : bool
+        Specifies whether Fejer-Scherliess model is used
+        If False, then 'exb.inp' is also archived
+    ExB_drifts : 10x2 ndarray of floats, string, or NoneType
+        Matrix of Fourier series coefficients dependent on solar local time
+        (SLT) in hours where
+        ExB_total = ExB_drifts[i,0] * cos((i + 1) * pi * SLT / 12)
```

Comment:
```suggestion
        exb_total = exb_drifts[i,0] * cos((i + 1) * pi * SLT / 12)
```

Reply:
at this point I realize if we want to make this change, then a find and replace is probably better.